pkg/controllers/jobset_controller.go

```diff
@@ -33,7 +35,10 @@ import (
 	jobset "sigs.k8s.io/jobset/api/v1alpha1"
 )
 
-const RestartsKey string = "jobset.sigs.k8s.io/restart-attempt"
+const (
+	RestartsKey       string = "jobset.sigs.k8s.io/restart-attempt"
+	parallelDeletions int    = 8
```

Comment:
isn't this too low? any idea what do we set this number to in other places like the Job API?

Reply:
> isn't this too low?

It makes sense. But I'm not sure appropriate value. How about 100 as a default value? Or, do you have any good idea?

> any idea what do we set this number to in other places like the Job API?

How about setting it via config: #55?

Reply:
Or, does that mean like https://github.com/kubernetes/kubernetes/blob/5fd29596ef699a8658c941c1c7cf7a45e796c1c6/pkg/controller/job/job_controller.go#L1458-L1466C63?

Reply:
I like the concept of a "slow start" in the linked code from the job controller. Maybe it would be simpler to start with some constant value like 100 then implement slow start separately?

Reply:
I guess for deletions we can continue to use a constant (I would set it to 50); but adopt a slow start approach for parallel creation.

Reply:
> I guess for deletions we can continue to use a constant (I would set it to 50); but adopt a slow start approach for parallel creation.

Great suggestions! I agree.